website/docs/workspaces/multi-tenancy.mdx

```diff
@@ -110,13 +115,23 @@ For more control on a specific tenant a tenancy file should be used, the command
 gitops create tenants --name test-tenant --namespace test-ns1 --namespace test-ns2
 ```
 
-The above will create the namespaces and permissions through a `ServiceAccount` with the same name as the tenant in that case `test-tenant` in each required namespace.
-The same can be done through a file as follows:
+<details><summary>Expand to view command output</summary>
 
 ```bash
-gitops create tenants --from-file tenants.yaml
+namespace/test-ns1 created
+test-ns1/serviceaccount/test-tenant created
+test-ns1/rolebinding.rbac.authorization.k8s.io/test-tenant-service-account-cluster-admin created
+namespace/test-ns2 created
+test-ns2/serviceaccount/test-tenant created
+test-ns2/rolebinding.rbac.authorization.k8s.io/test-tenant-service-account-cluster-admin created
+policy.pac.weave.works/weave.policies.tenancy.test-tenant-allowed-application-deploy created
 ```
 
+</details>
+
+The above will create the namespaces and permissions through a `ServiceAccount` with the same name as the tenant in that case `test-tenant` in each required namespace.
+The same can be done through a file as follows:
+
```

Comment:
133 and 135 say the same thing, so we can remove one. probably 135

Reply:
addressed and removed 135.

I've tested all the instructions and from my point of view it can fit ok  :+1:, if you have any feedback or something I missed, please lemme know

Reply:
So long as you believe the technical instructions are good, then that is enough for me 👍

Reply:
perfect